File: flutter-main/flutter-main/OOP/p10.dart
class Demo{
	final int? x;
	final String? str;
	const Demo(this.x,this.str);
}
void main(){
	Demo obj1 = const Demo(10,"ghefge");
	print(obj1.hashCode);
	Demo obj2 = const Demo(155,"ghefge");
	print(obj2.hashCode);
}

File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/struct/p10.c
//object initiliazation...method 1...

#include<stdio.h>
	struct movie{
		char mName[20];
		int noofticket;
		float price;
	}obj1={"kantara",2,300.30};

void fun(){

	struct movie obj2={"Tiger3",2,400};
	printf("%s\n",obj3.mName);
	printf("%d\n",obj3.noofticket);
	printf("%f\n",obj3.price);
}
void main(){
	struct Movie obj3={"Dishyrm",2,300};
	fun;

	printf("%s",obj1.mName);
	printf("%d",obj1.ticket);
	printf("%f",obj1.price);

	printf("%s",obj2.mName);
        printf("%d",obj2.ticket);
        printf("%f",obj2.price);
}




		

